api/task_run.go

```diff
@@ -45,6 +45,18 @@ type TaskRunResultPayload struct {
 	Version     string `json:"version,omitempty"`
 }
 
+// TaskSchemaUpdateGhostSyncRunResultDetail is the detail for a gh-ost sync task.
```

Comment:
Can we implement it as generic task progress to be used by other features? We may only need percentage and ETA in Unix timestamp.